plugins/node/opentelemetry-instrumentation-mysql/test/index.test.ts

```diff
@@ -154,6 +155,89 @@ describe('mysql@2.x-Tracing', () => {
     });
   });
 
+  describe('enhancedDatabaseReporting:true config, should track mysql.values', () => {
+    before(() => {
+      instrumentation.disable();
+      const config: MySQLInstrumentationConfig = {
+        enhancedDatabaseReporting: true,
+      };
+      instrumentation.setConfig(config);
+      instrumentation.enable();
+    });
+
+    after(() => {
+      instrumentation.disable();
+      instrumentation.setConfig({});
+      instrumentation.enable();
+    });
+
+    it('call conn.query(sqlString) with no values', done => {
+      const span = provider.getTracer('default').startSpan('test span');
+      context.with(trace.setSpan(context.active(), span), () => {
+        const query = connection.query(
+          'SELECT * FROM `books` WHERE `author` = "David"'
+        );
+        query.on('end', () => {
+          const spans = memoryExporter.getFinishedSpans();
+          assert.strictEqual(
+            spans[0].attributes[AttributeNames.MYSQL_VALUES],
+            ''
+          );
```

Comment:
If there are no values, should we record this attribute?

Reply:
good question that I asked myself too. To my opinion, the answer is yes, because otherwise, if we don't present this, the user might think there is a problem (because he doesn't see the parameter at all). What do you think?